src/components/content-tab-assistant.tsx

```diff
@@ -219,6 +219,37 @@ function Anchor( props: JSX.IntrinsicElements[ 'a' ] & ExtraProps ) {
 	);
 }
 
+const UsageLimitReached = () => {
+	const { daysUntilReset } = usePromptUsage();
+
+	// Determine if the reset is today
```

Comment:
For this bit, I have done the following: if the days are more than 1, then we say 1 day, 2 days etc as the design indicates. If it is less than 24 hours, I am displaying "today". I did not see any specification about this in the design.

One thing we can also do is to use a countdown that is similar to the one for expiring demo sites but it might be too much. Open to suggestions